api/envoy/extensions/common/ratelimit/v3/ratelimit.proto

```diff
@@ -121,8 +123,20 @@ message RateLimitDescriptor {
 
   // Optional rate limit override to supply to the ratelimit service.
   RateLimitOverride limit = 2;
+
+  // Optional hits_addend for the rate limit descriptor. If set the value will override the
+  // request level hits_addend.
+  // [#not-implemented-hide:]
+  google.protobuf.UInt32Value hits_addend = 3;
```

Comment:
In the request level, it can be an integer or a format string. How come here it is only an integer?

Reply:
The format string is used to extract a number based on the request and stream info. See the comments of `format` string comment

```
    // Substitution format string to extract the number of hits to add to the rate limit descriptor.
    // The same :ref:`format specifier <config_access_log_format>` as used for
    // :ref:`HTTP access logging <config_access_log>` applies here.
    //
    // .. note::
    //   The format string must contain only single valid substitution field that will be replaced
    //   with a non-negative number.
    //
    // For example, the ``%BYTES_RECEIVED%`` format string will be replaced with the number of bytes
    // received in the request.
```